build.sbt

```diff
@@ -179,6 +181,26 @@ lazy val `java-trace` = project
       "io.opentelemetry" % "opentelemetry-sdk-testing" % OpenTelemetryVersion % Test,
       "org.typelevel" %%% "cats-effect-testkit" % CatsEffectVersion % Test,
       "co.fs2" %% "fs2-core" % FS2Version % Test
+    ),
+    mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$Runner"
+      ),
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$Runner$"
+      ),
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$TimestampSelect"
+      ),
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$TimestampSelect$"
+      ),
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$TimestampSelect$Delegate$"
+      ),
+      ProblemFilters.exclude[MissingClassProblem](
+        "org.typelevel.otel4s.java.trace.SpanBuilderImpl$TimestampSelect$Explicit$"
+      )
```

Comment:
None of this code would be inlined by the macros, right?  This just bit Circe on Scala 3...

I have a feeling our next release will be 0.2, but as long as we're not changing the public API, this is a noble effort.

Reply:
No, these classes shouldn't appear in the inline positions